tutorial/aws-serverless-demo/SkierApiFunction/src/main/java/sql/HikariDS.java

```diff
@@ -0,0 +1,55 @@
+package sql;
+
+import com.zaxxer.hikari.HikariDataSource;
+import java.sql.Connection;
+import java.sql.SQLException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class HikariDS {
```

Comment:
This class still strikes me as really wasteful given that only one request will be running on a single lambda instance at any given time.

Reply:
I agree! Your approach makes more sense. Let's mention this in our presentation.